online-boutique/kustomize/velero-backup.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: velero.io/v1
+kind: Schedule
+metadata:
+  name: online-boutique-backup
+  namespace: k8saas-system
```

Comment:
This is so sad :( Let's hope this issue gets some more traction in the coming months https://github.com/vmware-tanzu/velero/issues/18